src/index.ts

```diff
@@ -213,19 +214,65 @@ server.tool(
   },
 );
 
+server.tool(
+  "get_follower_ids",
+  "Gets the list of User IDs of users who have added your LINE Official Account as a friend.",
+  {
+    start: z
+      .string()
+      .optional()
+      .describe(
+        "Continuation token for pagination. If omitted, fetches from the beginning.",
+      ),
+    limit: z
+      .number()
+      .int()
+      .min(1)
+      .max(1000)
+      .optional()
+      .describe(
+        "Maximum number of user IDs to retrieve (1-1000). Default is 1000.",
+      ),
```

Comment:
```suggestion
      .default(300)
      .describe(
        "Maximum number of user IDs to retrieve (1-1000). Default is 300.",
      ),
```
Auctually, the dafault limit is 300.
https://developers.line.biz/en/reference/messaging-api/#get-follower-ids-query-parameters
In addition, how about using default() instead of optional() ?